client/app/components/routes/access-requests/form/Review.tsx

```diff
@@ -0,0 +1,108 @@
+"use client";
+
+import { Alert, Button, ButtonGroup, Box } from "@mui/material";
+import RecommendIcon from "@mui/icons-material/Recommend";
+import DoNotDisturbIcon from "@mui/icons-material/DoNotDisturb";
+import { actionHandler } from "@/app/utils/actions";
+import React, { useState } from "react";
+import { Status } from "@/app/types";
+import { UserOperatorFormData } from "@/app/components/form/formDataTypes";
+
+interface Props {
+  userOperator: UserOperatorFormData;
+  userOperatorId: number;
+}
+
+export default function Review(props: Props) {
+  const [errorList, setErrorList] = useState([] as any[]);
+  const [successMessageList, setSuccessMessageList] = useState([] as any[]);
+
+  async function approveRequest() {
+    props.userOperator.status = Status.APPROVED;
+    const response = await actionHandler(
+      `registration/user-operators/${props.userOperatorId}/update-status`,
+      "PUT",
+      `dashboard/user-operators/${props.userOperatorId}`,
+      {
+        body: JSON.stringify(props.userOperator),
+      },
+    );
+    if (response.error) {
+      setErrorList([{ message: response.error }]);
+    } else if (response.ok) {
+      setSuccessMessageList([
+        { message: "You have approved the request for prime admin access." },
+      ]);
+
+      // Add a delay of 2000 milliseconds (2 seconds) before clearing the success message
```

Comment:
I don't see the success/reject message when I click the button (though I do see the change in the db): https://www.loom.com/share/86988548fef04422bbed9da5bb65c36a
Have I missed a step somewhere?
Also, I think this may be another case of changing wireframes, because it looks like the message is supposed to stay on the page now

Reply:
I don't know what change I made caused it not to persist.